lib/cursor.js

```diff
@@ -109,22 +109,48 @@ module.exports = {
       launder: function(s) {
         s = self.apos.launder.string(s);
 
-        if (!s.match(/^\d\d\d\d\-\d\d$/)) {
+        if (!s.match(/^\d\d\d\d-\d\d$/)) {
           return null;
         }
 
         return s;
       },
       choices: function(callback) {
-        return self.clone().upcoming(null).toDistinct('startDate', function(err, results) {
+        return self.clone().upcoming(null).toArray(function(err, results) {
           if (err) {
             return callback(err);
           }
-          return callback(null, _.uniq(_.each(results, function(value, key) { results[key] = value.substr(0,7) })).sort().reverse());
+
+          var months = [];
+
+          _.forEach(results, function (result) {
```

Comment:
_.each is succinct.

Reply:
I chose `forEach` because `each` is now an alias for it, but I can use that if you prefer.